content/content/oop_macro.md

```diff
@@ -206,6 +209,11 @@ class Dog of Animal:
 class Cat of Animal:
```

Comment:
Also, update the comment at https://github.com/flaviut/nim-by-example/pull/45/files#diff-cb8c0be3e9f135b42d69c85f323a317cR184